src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -81,6 +81,11 @@
 						@click.stop="handlePrivateReply">
 						{{ t('spreed', 'Reply privately') }}
 					</NcActionButton>
+					<NcActionButton icon="icon-clippy"
+						close-after-click
+						@click.stop="handleCopyMessageText">
+						{{ t('spreed', 'Copy message text') }}
```

Comment:
Only found it in hackerone to compare, there they call it "Copy markdown":
![grafik](https://github.com/nextcloud/spreed/assets/213943/2462e3bf-4904-4225-a8e7-a92b12edb787)


I'm a bit afraid of "Copy message text" and "Copy message link" being right next to each others with soo much shared label text.

Reply:
Generated some more options :robot: 
I'd like to stick with something similar to p3 or p10, as Markdown will be hard to translate for other languages

1. Copy Markdown Text
2. Copy Text with Markdown
3. Copy Formatted Text
4. Copy Markdown Content
5. Copy Message with Formatting
6. Copy Styled Text
7. Copy Text and Markup
8. Copy Rich Text
9. Copy Text as Is
10. Copy Text with Formatting

Reply:
cc some @nextcloud/designers on it

Reply:
I'd use 1